sw/device/lib/testing/random.h

```diff
@@ -0,0 +1,32 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOM_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOM_H_
+
+#include <stdint.h>
+
+/**
+ * Generate a pseudo-random unsigned integer.
+ *
+ * For testing purposes, a simple LFSR generates random numbers starting with
+ * a known seed.
+ * @return A pseudo-random 32-bit value.
+ */
+uint32_t random_gen32(void);
+
+/**
+ * Generate a random unsigned integer within a given range.
+ *
+ * This function invokes `urandom()` and restricts the returned value to be
```

Comment:
I think you should update this to `random_gen32()`.

Reply:
Done.